### PR TITLE
add get started and overview section for docs

### PR DESCRIPTION
**Pull request by ehborisov**

### CODE REVIEW

File: docs/2.0/overview.adoc [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+Timeline tab visualizes retrospective of tests execution, allure adaptors collect precise timings of tests,
+and here on this tab they are arranged accordingly to their sequential or parallel timing structure.
+
+image::overview_timeline.png["Timeline tab overview"]
+
+=== Behaviors
+
+For Behavior-driven approach, this tab groups test results according to Feature and Story tags.
+
+image::overview_behaviors.png["Behaviors tab overview"]
```
Comment (eroshenkoam) [inline docs/2.0/overview.adoc]: rename image to behaviors-tab.png

File: docs/2.0/overview.adoc [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+severity and duration diagrams.
+
+image::overview_dashboards.png["Dashboards tab overview"]
+
+=== Timeline
+
+Timeline tab visualizes retrospective of tests execution, allure adaptors collect precise timings of tests,
+and here on this tab they are arranged accordingly to their sequential or parallel timing structure.
+
+image::overview_timeline.png["Timeline tab overview"]
```
Comment (eroshenkoam) [inline docs/2.0/overview.adoc]: rename image to timeline-tab.png

File: docs/2.0/overview.adoc [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+On the xUnit tab a standard structural representation of executed tests, grouped by suites and classes can be found.
+
+image::overview_xunit.png["X Unit tab"]
+
+=== Graph
+
+Graphs allow you to see different statistics collected from the test data: statuses breakdown or
+severity and duration diagrams.
+
+image::overview_dashboards.png["Dashboards tab overview"]
```
Comment (eroshenkoam) [inline docs/2.0/overview.adoc]: rename image to graph-tab.png

File: docs/2.0/overview.adoc [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+Home page widgets are draggable and configurable. Also, Allure supports it's own plugin system, so quite different
+ widget layouts are possible.
+
+Navigation bar is collapsible and enables you to switch into several of the basic results overview modes.
+
+=== xUnit
+
+On the xUnit tab a standard structural representation of executed tests, grouped by suites and classes can be found.
+
+image::overview_xunit.png["X Unit tab"]
```
Comment (eroshenkoam) [inline docs/2.0/overview.adoc]: rename image to xunit-tab.png

File: docs/2.0/overview.adoc [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+Each Allure report is backed by a tree-like data structure, that represents a test execution process. Different tabs
+allow to switch between the views of the original data structure thus giving a different perspective.
+Note that all tree-like representations including Behaviors, Categories, xUnit and Packages support filtering and
+are sortable.
+
+=== Overview page
+
+Entry point for every report would be the 'Overview' page with dashboards and widgets:
+
+image::overview_report_overview.png["Report overview"]
```
Comment (eroshenkoam) [inline docs/2.0/overview.adoc]: rename image to overview-tab.png

File: docs/2.0/overview.adoc [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+For Behavior-driven approach, this tab groups test results according to Feature and Story tags.
+
+image::overview_behaviors.png["Behaviors tab overview"]
+
+=== Packages
+
+Packages tab represents a tree-like layout of test results, grouped by different packages.
+
+image::overview_packages.png["Packages tab overview"]
```
Comment (eroshenkoam) [inline docs/2.0/overview.adoc]: rename image to packages-tab.png

File: docs/2.0/overview.adoc [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+=== Test case page
+
+From some of the results overview pages described above you can go to the test case page after clicking on the
+individual tests. This page will typically contain a lot of individual data related to the test case: steps
+executed during the test, timings, attachments, test categorization labels, descriptions and links.
+
+image::overview_test_case.png["Test case result page"]
```
Comment (eroshenkoam) [inline docs/2.0/overview.adoc]: rename image to testcase.png

File: docs/2.0/getstarted.adoc [hunk 8]
```diff
@@ -0,0 +1,73 @@
+= Get Started
+
+To generate your first report you will need to go through just a few simple steps:
+
+ * Downloading and installing Allure commandline application suitable for your environment.
+ * Locating test execution data that you have to build a report on.
```
Comment (eroshenkoam) [inline docs/2.0/getstarted.adoc]: May be we can split step into two: 
* launch your tests
* build report on

File: docs/2.0/getstarted.adoc [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+== Installing a commandline
+Several options for Allure installation are currently supported:
+
+=== Linux
+For debian-based repositories a PPA is provided:
+[source, bash]
+----
+sudo apt-add-repository ppa:yandex-qatools/allure-framework
```
Comment (baev) [inline docs/2.0/getstarted.adoc]: why yandex-qatools

File: docs/2.0/getstarted.adoc [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+brew install allure-commandline
+----
+
+=== Windows
+Windows and other Unix distributions require manual installation:
+
+. Download the latest version as zip archive from link:https://github.com/allure-framework/allure2/releases[releases page].
+. Unpack the archive to allure-commandline directory.
+. Navigate to *bin* directory.
+. Use *allure.bat* for Windows or *allure* for other Unix platforms.
```
Comment (baev) [inline docs/2.0/getstarted.adoc]: Windows section and `other Unix platforms`

**Pull request merged**